charts/kibana/templates/ingress.yaml

```diff
@@ -21,6 +21,8 @@ metadata:
     nginx.ingress.kubernetes.io/auth-url: https://houston.{{ .Values.global.baseDomain }}/v1/authorization
     nginx.ingress.kubernetes.io/auth-signin: https://app.{{ .Values.global.baseDomain }}/login
     nginx.ingress.kubernetes.io/auth-response-headers: authorization, username, email
+    nginx.ingress.kubernetes.io/configuration-snippet: |
+      more_set_headers "Content-Security-Policy: script-src 'self'";
```

Comment:
this config snipet is required by kibana after migrating to 8.x due to strict requirement in CSP rules